src/scripts/fetch_package_names.js

```diff
@@ -1,58 +1,67 @@
 #!/usr/bin/env node
 // ./fetch_package_names <suffix> > output
 // ./fetch_package_names "-loader" > output.json
-const GitHubApi = require("github");
+const GitHubApi = require('github');
 
 if (require.main === module) {
-    main();
+  main();
 } else {
-    module.exports = fetchPackageNames;
+  module.exports = fetchPackageNames;
 }
 
 function main() {
   const organization = process.argv[2];
   const suffix = process.argv[3];
 
-  if(!organization) {
+  if (!organization) {
     return console.error('Missing organization!');
   }
-  if(!suffix) {
+  if (!suffix) {
     return console.error('Missing suffix!');
   }
 
-  fetchPackageNames({
-    organization: organization,
-    suffix: suffix
-  }, function(err, d) {
-    if (err) {
-      return console.error(err);
-    }
+  fetchPackageNames(
+    {
+      organization: organization,
+      suffix: suffix
+    },
+    function(err, d) {
```

Comment:
can you define the error handler outside of function call and just pass it here pls? This would make it even more readable, same goes for following error handler

Reply:
Out of the scope of this PR, I just reformatted that file, actually I should revert it.
Also, we should change how fetch script works, one is reading stdout from the previous one, making it harder to test in isolation.

Reply:
nah dont revert, formatting is good now